src/compiler/diagnosticMessages.json

```diff
@@ -1493,6 +1493,10 @@
         "category": "Message",
         "code": 1476
     },
+    "Instantiation expression cannot be followed by property access.": {

```

Comment:
Should this instead complain on the type arguments and say they should be removed? And quickfix?